tools/lint-hotkeys

```diff
@@ -91,6 +91,24 @@ def generate_hotkeys_file() -> None:
     print(f"Hot Keys list saved in {OUTPUT_FILE}")
 
 
+def is_every_key_category_valid() -> bool:
+    """
+    Check for typos in key categories in KEYS_FILE
+    """
+    error_flag = False
+    for key, binding in KEY_BINDINGS.items():
+        key_category = binding.get("key_category")
+        if key_category not in HELP_CATEGORIES:
+            print(
+                f"Invalid key_category '{key_category}' for key '{key}'."
+                f" Choose a category from:\n{', '.join(HELP_CATEGORIES.keys())}\n"
```

Comment:
NIT: For a dev tool this isn't as important, but with this number of categories we already run quite far to the right, so a vertical list may be cleaner to read.